proposals/serverlessworkflow.md

```diff
@@ -0,0 +1,170 @@
+# Serverless Workflow
+
+**Name of project**: Serverless Workflow
+
+**Description**:
+
+TOC created the Serverless Working Group to investigate
+the Serverless landscape. The outputs of the WG included:
+- a [whitepaper]https://github.com/cncf/wg-serverless/blob/master/whitepapers/serverless-overview/README.md) that:
+  - defines Serverless and its terminology
+  - describes common use cases for the technology
+  - compares it with other Cloud Native technologies and \*aaS environments
+  - describes the common architecture of Serverless platforms
+- a [landscape document](https://docs.google.com/spreadsheets/d/10rSQ8rMhYDgf_ib3n6kfzwEuoE88qr0amUPRxKbwVCk/edit#gid=0)
+  that lists well-known open-source and proprietary Serverless platforms
+  and tools
+- a set of recommended next steps for the WG, as part of the whitepaper:
+  - encourage more Serverless technology vendors and open source developers
+    to join the CNCF
+  - foster an open ecosystem by establishing interoperable APIs, in particular
+    around: Events, Deployments and Workflows
+  - provide additional education as needed
+
+In addition to CloudEvents specification, the Serverless WG has agreed to form a sub working group to work on
```

Comment:
Since you didn't mention CE before this reference it might ease the transition a bit if you added something like:
```
The Working Group's first focus area was around making it easier for middleware to process Events, which led to the development of the [CloudEvents](https://cloudevents.io) specification. Additionally, the Serverless WG...
```

Reply:
good point. added.